### ISSUE TITLE
Secondary button styles missing in Light HC

### ISSUE THREAD

**Issue by misolori** [microsoft/vscode#144608]
<img width="558" alt="CleanShot 2022-03-07 at 08 47 19@2x" src="https://user-images.githubusercontent.com/35271042/157079229-faeb9c33-12aa-46ae-870a-04c3ef809ff1.png">

<img width="481" alt="CleanShot 2022-03-07 at 08 47 57@2x" src="https://user-images.githubusercontent.com/35271042/157079269-9a9f46a3-b0d8-4164-a464-06c16aef0170.png">

Compared to Dark HC

<img width="543" alt="CleanShot 2022-03-07 at 08 48 41@2x" src="https://user-images.githubusercontent.com/35271042/157079436-c76cbe60-95fe-456c-95c1-32424eeca344.png">

<img width="481" alt="CleanShot 2022-03-07 at 08 49 00@2x" src="https://user-images.githubusercontent.com/35271042/157079458-9b4901b9-9429-4220-a107-58ccb009efbc.png">

**Comment by misolori** [comment]
Ah yea, will close as a dup. Thanks!
